apps/restconf/restconf_root.c

```diff
@@ -441,7 +490,41 @@ api_root_restconf(clicon_handle h,
     }
     else if (strcmp(api_resource, "data") == 0){ /* restconf, skip /api/data */
 	if (api_data(h, req, path, pcvec, 2, qvec, indata,
-		     pretty, media_out) < 0)
+		     pretty, media_out, IETF_DS_NONE) < 0)
+	    goto done;
+    }
+    else if (strcmp(api_resource, "ds") == 0) {
```

Comment:
I have a basic/stupid? question on how the "ds" resource is specified and accessed in the web server, eg nginx or evhtp code.
Why is this a separate data path than the "data" resource?

Reply:
This is the "new way" of specifying datastores in RESTconf as per https://tools.ietf.org/html/rfc8527 and https://tools.ietf.org/html/rfc8342. AFAIK the reason that no web server change is required is because the entire request is encoded in XML as an rpc and passed transparently to the underlying netconf layer.

Reply:
Yes, I missed that clause in RFC8572, my mistake